lib/app_server.py

```diff
@@ -158,7 +158,7 @@ def deploy(self, vardir=None, silent=True, need_init=True):
             warn_unix_socket(path)
             self.iproto = path
         else:
-            self.iproto = str(find_port())
+            self.iproto = self.localhost + ':0'
```

Comment:
AFAIR it is only needed for the `create server` command with the `return_listen_uri` option. Now it'll return an unsuitable result.

Feel free to drop this feature together with the `self.iproto` field if you don't see a real need, but, please, don't leave it broken.

Reply:
No, it is not used for the `create server` command with the `return_listen_uri` option. The `create server` command uses `TarantoolServer` (please check [this](https://github.com/tarantool/test-run/blob/b59464f17f391ecffd985a2366777bc3cb354eca/lib/app_server.py#L54-L57)).

Actually, this is just an attribute that is used only [here](https://github.com/tarantool/test-run/blob/b59464f17f391ecffd985a2366777bc3cb354eca/lib/app_server.py#L36). I guess, the line `os.putenv("LISTEN", server.iproto)` exists only because some app tests have `box.cfg {listen = os.getenv('LISTEN')}` inside the test script. 

I can remove this attribute and move this `if ... else ...` block to the `run_server` function.

Reply:
Or rename it into `self.listen` to not be confused.

Reply:
Fixed.